vpr/src/route/rr_graph_storage.cpp

```diff
@@ -739,3 +739,53 @@ t_rr_graph_view t_rr_graph_storage::view() const {
         vtr::make_const_array_view_id(edge_dest_node_),
         vtr::make_const_array_view_id(edge_switch_));
 }
+
+void t_rr_graph_storage::reorder(const vtr::vector<RRNodeId, RRNodeId>& order,
```

Comment:
Should comment what this routine does, and exactly what order and inverse_order are (entry
order[old_rr_id] = new_rr_id and so on).

Reply:
Done.